WtWSMS/common/landed_titles/BP_e_britannia.txt

```diff
@@ -3110,8 +3850,6 @@ e_britannia = {
 				b_evesham = {
 					province = 1625
 					cultural_names = {
-					    #Celtic Cultures
-					    name_list_briton = cn_cair_afon
```

Comment:
Why the removal?

Reply:
> Why the removal?

Not really sure why I did this, re-added.

Reply:
> > Why the removal?
> 
> Not really sure why I did this, re-added.

Now I remember why I did this. I've added some Welsh translations of English city names from an obscure source, but that name was completely made up. Evesham means "land in a river-bend belonging to a man called Ēof", and Cairafon means "ford by the rivel Avon".